tests/callbacks/test_model_checkpoint.py

```diff
@@ -97,18 +97,56 @@ def test_model_checkpoint_no_extraneous_invocations(tmpdir):
     assert 1 == result
 
 
+def test_model_checkpoint__format_checkpoint_name():
+    # empty filename:
+    r = ModelCheckpoint._format_checkpoint_name('', 3, {})
+    assert r == 'epoch=3'
```

Comment:
can we have better var names than "r" or "fp"?

Reply:
also, how will the checkpoint be formatted with save_last=True?

Reply:
Updated var names.

What do you mean? There is no `save_last` parameter in the `_format_checkpoint_name` function